pkg/acsengine/const.go

```diff
@@ -67,6 +67,8 @@ const (
 	DefaultKubernetesCloudProviderRateLimitQPS = 3.0
 	// DefaultKubernetesCloudProviderRateLimitBucket is 10, takes effect if DefaultKubernetesCloudProviderRateLimit is true
 	DefaultKubernetesCloudProviderRateLimitBucket = 10
+	// DefaultTillerImage defines the Helm Tiller deployment version on Kubernetes Clusters
+	DefaultTillerImage = "tiller:v2.5.1"
```

Comment:
Ohh you fancy! Seriously this will be nice to refactor the other string settings into consts in future work.